client/src/components/ReportCalendar.js

```diff
@@ -8,6 +8,8 @@ import moment from "moment"
 import PropTypes from "prop-types"
 import React, { useRef } from "react"
 import { useHistory } from "react-router-dom"
+import ReactDOM from "react-dom"
+import PlanningConflictForReport from "components/PlanningConflictForReport"
```

Comment:
Please properly order the imports (alphabetically on imported package if there's no command for it in your IDE).